framework/resources/Functional/table_function/positive/table_function.json

```diff
@@ -21,6 +21,11 @@
          "mode": "cp",
          "src": "Datasources/table_function",
          "dest": "table_function/"
+      },
+      {
+        "mode": "gen",
+        "src": "Datasources/table_function/DRILL-5166_generate_data.sh",
```

Comment:
Looks like your script copies generated file to dfs. But would the section "mode": "cp" does the same if your script put the generated file in the same "src" dir?

Reply:
"mode": "cp" is executed prior to the "mode": "gen", so the generated files won't be copied by the "cp" section at the same test execution.

Reply:
True. If we pre-generate the data and reuse it, then we can rely on "cp" mode. The generated data needs to be put on s3. Let us know if you need help on that. This is our current practice how we handle large dataset. Thanks.